CHANGES.rst

```diff
@@ -13,6 +13,39 @@ Changelog
 
 .. towncrier release notes start
 
+3.16.2 (2021-12-22)
+===================
+
+
+Bugfixes
+--------
+
+- Fixed sync of repositories using 'sha' as an alias for the sha1 checksum-type.
+  (backported from #9580)
+  `#9624 <https://pulp.plan.io/issues/9624>`_
```

Comment:
Unfortunately we have go fix up all the places where it says "backported from" and the issue numbers